### PR TITLE
now config

### PR DESCRIPTION
**Pull request by plusbpm**

### CODE REVIEW

File: config/webpack/config.common.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
 /* eslint-disable import/no-extraneous-dependencies */
-require('dotenv').config();
+const path = require('path');
+const dotenv = require('dotenv');
 const { EnvironmentPlugin } = require('webpack');
 const pick = require('lodash/pick');
 
+const envFilename = process.env.IS_NOW === 'true' ? '.env.now' : '.env';
```
Comment (droganov) [inline config/webpack/config.common.js]: Why true is string?

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "dev": "next -p 8000",
     "now:dev": "now dev -p 8000",
     "build": "next build",
+    "now-build": "npm run build",
```
Comment (droganov) [inline package.json:16]: Why do we need this?

**Review by droganov — CHANGES_REQUESTED** [review]

File: config/webpack/config.common.js (hunk 1 above)
Comment (plusbpm) [reply config/webpack/config.common.js]: in Environment all is string

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,5 @@
 .env
-.env.build
+.env.now
```
Comment (droganov) [inline .gitignore]: Did you create .env.now.sample?

**Review by droganov — CHANGES_REQUESTED** [review]

File: now.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "alias": "rkta-starter-apollo",
+  "build": {
+    "env": {
+      "IS_NOW": "true"
```
Comment (droganov) [inline now.json]: move to .env.now

**Review by droganov — CHANGES_REQUESTED** [review]

**Pull request merged**